File: src/lib/pjecalc/core/comum/validators/calculo/informacao.ts
/**
 * PJe-Calc v2.15.1 — Informacao
 * Porte de: br.jus.trt8.pjecalc.negocio.comum.validators.calculo.Informacao
 */
import { MsgValidador } from './msg-validador';

export class Informacao extends MsgValidador {
  private static readonly TIPO = 'INFO';
  isImpeditivo(): boolean { return false; }
  getTipo(): string { return Informacao.TIPO; }
}

File: pjecalc-fonte/negocio/br/jus/trt8/pjecalc/negocio/comum/validators/calculo/Informacao.java
/*
 * Decompiled with CFR 0.152.
 */
package br.jus.trt8.pjecalc.negocio.comum.validators.calculo;

import br.jus.trt8.pjecalc.negocio.comum.validators.calculo.MsgValidador;

public class Informacao
extends MsgValidador {
    private static final long serialVersionUID = 3494106834486138465L;
    private static final String TIPO = "INFO";

    @Override
    public boolean isImpeditivo() {
        return false;
    }

    @Override
    public String getTipo() {
        return TIPO;
    }
}

